SDWebImage/SDWebImageDownloader.m

```diff
@@ -438,7 +438,7 @@ - (void)dealloc {
 - (instancetype)initWithDownloadOperation:(NSOperation<SDWebImageDownloaderOperation> *)downloadOperation {
     self = [super init];
     if (self) {
-        _downloadOperation = downloadOperation;
+        self.downloadOperation = downloadOperation;
```

Comment:
Why we need to change this to getter? I think we should always keep ivar in `init` method? 🤔

Reply:
Maybe use `_ivar` is better...I'm not a developer from MRC time, I just think, the setter of property should always been called, instead of directlly acess ivar. Like you use `copy` annotation or write a custom setter method like lazy initilizer. But however, in the init method of Cbjective-C, maybe use ivar is OK because it's called init method and provide the default state.